invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +128,29 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+.deposits-community-header {
+  margin: -20px 0px 20px 0px;
+  background-color: #ededed;
+  height: 65px;
+}
+
+.community-header {
+  padding-top: 10px;
+  &.empty {
+    padding-top: 23px;
+  }
+}
+
+.ui.container.community-header {
+  .ui.mini.image {
```

Comment:
what is the problem with the default semantic ui image class?

Reply:
The issue is that it's not a square.
### Default image
![image](https://user-images.githubusercontent.com/15194802/154066766-ed41c43b-6fa2-4739-aaff-c3619cefb923.png)
### Our custom css
![image](https://user-images.githubusercontent.com/15194802/154066729-bc1cf466-40de-4f8d-9da7-f231df61f820.png)

Reply:
I think the component is ready so I would pass a prop with className to the image component created by Zach and then define this special class here, instead of overwriting the semantic ui class